core-concepts/services/consumers.md

```diff
@@ -10,7 +10,7 @@ A consumer continually reads messages from a single stream and dispatches them t
 
 Many consumers can be hosted together in a single service, allowing a component to be fed messages from many streams.
 
-In addition, consumers keep track of its reader's progress through a stream, and controls polling rates, and pre-fetching batches of messages, and the dispatching of messages to handlers.
+In addition, a consumer keeps track of its reader's progress through a stream, and controls polling rates, and pre-fetching batches of messages, and the dispatching of messages to handlers.
```

Comment:
regardless to whether this change is correct `its` should be replaced with `it's`

Reply:
Are you sure about that? I think `its` refers to the ownership, while `it's` would simply mean `it is`